rules/solid/tests/Rector/Foreach_/ChangeNestedForeachIfsToEarlyContinueRector/Fixture/multi_exprs_with_OR_both_truthy.php.inc

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Rector\SOLID\Tests\Rector\Foreach_\ChangeNestedForeachIfsToEarlyContinueRector\Fixture;
+
+use Rector\Core\Testing\Contract\RunnableInterface;
+
+class MultiExprsWithOrBothTruthy implements RunnableInterface
+{
+    public function run()
+    {
+        $partPackageList = [ new \StdClass(), 'string' ];
+        $payload = [];
+
+        foreach( $partPackageList as $partPackage )
+        {
+            if(! is_null( $partPackage ) && ! is_object($partPackage) )
+            {
+                if (1 === 1 || 2 === 2)
```

Comment:
I think this `if` is enough to reproduce. The `if` above is not needed. Am I right?

Reply:
ChangeNestedForeachIfsToEarlyContinueRector requires at least two levels of ifs to activate when I tried it in Rector v0.7.34

Reply:
rules/solid/src/Rector/Foreach_/ChangeNestedForeachIfsToEarlyContinueRector.php: L#99

```
    public function refactor(Node $node): ?Node
    {
        $nestedIfsWithOnlyNonReturn = $this->ifManipulator->collectNestedIfsWithNonBreaking($node);
        if (count($nestedIfsWithOnlyNonReturn) < 2) {
            return null;
        }

        return $this->processNestedIfsWithNonBreaking($node, $nestedIfsWithOnlyNonReturn);
    }
```

Reply:
I see. So `if ($value)` is enough, right?

Reply:
It's much less logic to work with than ` if(! is_null( $partPackage ) && ! is_object($partPackage) )`, where is like ~8 operations.

Reply:
Is this better?

Reply:
Much better, thanks!